ui/src/utils/updateUrl.ts

```diff
@@ -0,0 +1,48 @@
+import { Params } from '../common/params';
+import { SortByFields } from '../store/resource';
+import { titleCase } from '../common/titlecase';
+
+// This function returns all selected filters in a combination of params
+export const UpdateURL = (
+  search: string,
+  sort: string,
+  categories: string,
+  kinds: string,
+  catalogs: string
+) => {
+  //To get URLSearchParams object instance
+  const searchParams = new URLSearchParams(window.location.search);
+
+  //To delete query params if search is empty
+  if (!search && searchParams.has(Params.Query)) searchParams.delete(Params.Query);
+
+  //Sets query params
```

Comment:
```suggestion
  // Sets query params
```

Reply:
@pratap0007 can you please add spaces in other comments as well?